src/controllers/PostsController.ts

```diff
@@ -0,0 +1,21 @@
+import { Controller, Get } from '@tsed/common';
+import { Posts } from '../entities/Posts';
+
+import { PostsService } from '../services/PostsService';
+
+
+@Controller('/feed')
+export class UserController {
+	constructor(private postsService: PostsService) {}
+
+	@Get('/')
+	getPostsFeed(): Promise<Posts[]> {
+		return this.postsService.findAll();
+	}
+
+	/*
+	@Post('/subreddits/posts/create')
+	createPost(@BodyParams() posts: Posts): Promise<void> {
+		return this.postsService.create(posts);
+	}*/
```

Comment:
Preparation for future i guess but keep in mind that in apirary this endpoint shoud be /api/v1/subreddits/posts/create but now is under /api/v1/feed/subreddits/posts/create